CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorAPI/Models/ConversationAccount.cs

```diff
@@ -38,10 +39,12 @@ public ConversationAccount()
         /// Active Directory (AAD)</param>
         /// <param name="role">Role of the entity behind the account (Example:
         /// User, Bot, etc.). Possible values include: 'user', 'bot'</param>
-        public ConversationAccount(bool? isGroup = default(bool?), string id = default(string), string name = default(string), string conversationType = default(string), string role = default(string),  string aadObjectId = default(string))
+        /// <param name="tenantId">This conversation's tenant ID</param>
+        public ConversationAccount(bool? isGroup = default(bool?), string id = default(string), string name = default(string), string conversationType = default(string), string role = default(string), string aadObjectId = default(string), string tenantId = default(string))
```

Comment:
As we have recently found out (The hard way) adding additional parameters (even with defaults) like this actually breaks binary comparability. 

In this case, Can you add an overload instead, that takes the tenantId? This ctor should then call that ctor (with default(string)) and thus binary compatablity (as well as code compat) is preserved.

Reply:
Hrm. This is in a auto-generated class. Crap. 

Let's sync on this today, and see what we want to do.